packages/desktop/src/main/backendManager.ts

```diff
@@ -32,7 +26,7 @@ export const runBackend = async (): Promise<{
     agentHost: 'localhost',
     agentPort: options.socksPort,
     httpTunnelPort: options.httpTunnelPort,
-    io: dataServer.io,
+    socketIOPort: options.dataServerPort,
```

Comment:
Could you also change it in createApp in integration tests so we are consistent in using ConnectionsManager?